directory/src/main/java/org/ecclesia/directory/domain/Organization.java

```diff
@@ -3,21 +3,25 @@
 import javax.persistence.*;
 import javax.validation.constraints.NotBlank;
 import java.io.Serializable;
-import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 /**
  * Represents an organization. Data transfer object.
  */
 @Entity
 @Table(name = "organization")
-@Data
+@Getter @Setter @NoArgsConstructor
 public class Organization implements Serializable {
 
   private static final long serialVersionUID = 719423610411393862L;
 
   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
-  private long id;
+  private
+    long
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck
**Severity**: WARN
**File**: /var/jenkins_home/workspace/ecclesia_PR-6/directory/src/main/java/org/ecclesia/directory/domain/Organization.java L23

&#39;long&#39; has incorrect indentation level 4, expected level should be 6.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1939681163>*